.gitignore

```diff
@@ -14,3 +14,5 @@ obj/
 bin/
 installers/msi-language/Product.wxs
 node_modules
+zig-cache
```

Comment:
This is a rather awkward change given the history of the project. This would mean that the zig cache dir is ignored before it exists. The only reason it exists is because we jump around in (release) time. My suggestions in order of preference, but still 100% up to you, are 1. delete the directory, 2. ignore globally, 3. use `**/zig-cache/` as the ignore line.

Reply:
I'll go with option 1. Thanks.